packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/getuserinfo.md

```diff
@@ -1,28 +1,25 @@
-Display the user info by updating the `showUserInfo` function of `ContentView`:
+Display the user info by updating the `showUserInfo()` function of `ContentView.swift`:
 
 ```swift
 func showUserInfo() {
-  guard let authStateManager = authStateManager else {
-    showError(title: "Unable to Show User Info",
-              message: "No Okta client found.")
-
-    return
-  }
-
-  authStateManager.getUser() { response, error in
-    if let error = error {
-      showError(title: "Unable to Read User Info", error: error)
+   guard let credential = Credential.default,
+         let idToken = credential.token.idToken else {
```

Comment:
This can be simplified, since the `credential` variable isn't being used elsewhere in this function.

```suggestion
   guard let idToken = Credential.default?.token.idToken else {
```

Reply:
Nice catch... the dangers of updating functions for new ways of doing things and not paying attention to the whole thing... I used to use `credential` when it was a server call.  The change will be in the next commit.